vulnerabilities/models.py

```diff
@@ -3065,6 +3069,41 @@ def get_or_create_from_purl(self, purl: Union[PackageURL, str]):
 
         return package, is_created
 
+    def bulk_get_or_create_from_purls(self, purls: List[Union[PackageURL, str]]):
```

Comment:
Shall we have a boolean to mark if an impactd package is unfurled, so we do not unfurl that again